readme.md

```diff
@@ -31,10 +31,20 @@ foo();
 
 ## API
 
-### callerCallsite()
+### callerCallsite([options])
 
 Returns a [`callsite`](https://github.com/sindresorhus/callsites#api) object.
 
+#### options
+
+Type: `object`
+
+##### depth
+
+Type: `number`<br>
+Default: `0`
+
+The depth of callsite.
```

Comment:
It needs a better description than this. Users will ask themselves: "What exactly does depth mean in this context?". It would also be useful to include some use-cases.